lib/tasks/demo-build.js

```diff
@@ -88,29 +89,48 @@ function buildJs(buildConfig) {
 	}
 }
 
+function loadExternalDemoData(url) {
+	return request.get({
+		uri: url,
+		json: true
+	})
+	.then(data => {
+		return data;
+	});
+}
+
+function loadLocalDemoData(dataPath) {
+	return fileExists(dataPath)
+		.then(exists => {
+			if (exists) {
+				return readFile(dataPath, 'utf-8')
+					.then(file => {
+						try {
+							const fileData = JSON.parse(file);
+							if (typeof fileData === 'object') {
+								return fileData;
+							} else {
+								return {};
+							}
+						} catch (error) {
+							const e = new Error(`${dataPath} is not valid JSON.`);
+							e.stack = '';
+							throw e;
+						}
+					});
+			}
+		});
+}
+
 function loadDemoData(buildConfig) {
 	if (typeof buildConfig.demo.data === 'string') {
-		const dataPath = path.join(buildConfig.cwd, '/' + buildConfig.demo.data);
-		return fileExists(dataPath)
-			.then(exists => {
-				if (exists) {
-					return readFile(dataPath, 'utf-8')
-						.then(file => {
-							try {
-								const fileData = JSON.parse(file);
-								if (typeof fileData === 'object') {
-									return fileData;
-								} else {
-									return {};
-								}
-							} catch (error) {
-								const e = new Error(`${dataPath} is not valid JSON.`);
-								e.stack = '';
-								throw e;
-							}
-						});
-				}
-			});
+		const dataPathIsAbsolute = /^(?:https?:)\/\//.test(buildConfig.demo.data);
```

Comment:
Can we change the variable name from `dataPathIsAbsolute` to `dataPathIsHTTP` or `dataPathIsRemote`? Absolute makes me think that an absolute file-system path would set this variable to true (which it doesn't).